docs/41-managing-storage-via-R.md

```diff
@@ -0,0 +1,35 @@
+# Managing blob files in Azure Storage
+## Accessing your storage files through R
+Without installing Azure Storage Explorer or using the Azure Portal, users can access their resources through doAzureParallel wrapper functions around rAzureBatch's API calls.
+
+A container provides a grouping of a set of blobs. All blobs must be in a container. An account can contain an unlimited number of containers. A container can store an unlimited number of blobs. **Note** that the container name must be lowercase.
+
+Blob is a storage file of any type and size. The Azure Storage Blob service uses a flat storage scheme, not hierachical scheme.
```

Comment:
Kind of confusing first sentence. Please rephrase.

Also, you can point to Azure blob documentation for more info

Reply:
This is from the Azure blob documentation